app/src/main/res/values/strings.xml

```diff
@@ -631,6 +631,9 @@
   <string name="contactpatient_dx">Dx</string>
   <string name="contactpatient_visited">Visited</string>
   <string name="contactpatient_no_phone_number">No Phone Number</string>
+  <string name="contactpatient_diagnosis_diabetes">Diabetes</string>
+  <string name="contactpatient_diagnosis_hypertension">Hypertension</string>
+  <string name="contactpatient_diagnosis_none">None</string>
```

Comment:
I don't see any "None" option in designs or PRD. Can we confirm whether we need to show none or hide the diagnosis, please?

Reply:
Confirmed from the design team that we should have none [here](https://simpledotorg.slack.com/archives/C0259NK2L9X/p1625739390000300)